docs/low-code-spark/gems/machine-learning/ml-pinecone-lookup.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 3
+title: Pinecone Lookup
+id: ml-pinecone-lookup
+description: Lookup a vector embedding from a Pinecone Database
+tags: [generative-ai, machine-learning, llm, pinecone, openai]
+---
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/019qt3csq9?seo=false?videoFoam=true" title="Getting Started With SQL Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+The Pinecone Lookup Gem identifies content that is similar to a provided vector embedding. The Gem operation, input, and output are breifly described below.
```

Comment:
Could we put more spacing here? 

<img width="967" alt="Screenshot 2023-08-16 at 10 25 36 AM" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/3248329/4cc5bf5f-3d67-40d3-b941-2f8e60702bb0">